CoEpi/ui/healthquiz/HealthQuizViewModel.swift

```diff
@@ -30,7 +33,21 @@ class HealthQuizViewModel: UINotifier {
         notification = notificationSubject
             .asDriver(onErrorDriveWith: .empty())
 
-        observeSubmit()
+        let selectedSymptoms = questionsRelay
+            .map { questions in questions
+                .filter { $0.checked }
+                .map { $0.toSymptom() }
+            }
+        submitAction = Action { [symptomRepo] in
+            selectedSymptoms.flatMap {
```

Comment:
@i-schuetz This will result in one submission to backend (for all selected symptoms), right?

Reply:
Right :)